app/elements/account-elements/forgot-password-request-page.html

```diff
@@ -0,0 +1,54 @@
+<link rel="import" href="../../bower_components/paper-input/paper-input.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+<link rel="import" href="account-page.html">
+
+<dom-module id="forgot-password-request-page">
+  <template>
+    <account-page id="account" sub-route="/resetpassword" heading="Forgot password" body="[[body]]" on-iron-error="handleForgotPasswordError" on-iron-success="removeForgotPasswordMessage">
+      <div class="card-content">
+        <paper-input required auto-validate error-message="needs some text!"
+            label="Username" type="text" value="{{username}}"
+            on-keydown="removeForgotPasswordMessage"></paper-input>
+        <paper-input required auto-validate error-message="needs some text!"
+            label="Your email address" type="email" value="{{email}}"
+            on-keydown="removeForgotPasswordMessage"></paper-input>
+        <span class="errormessage" hidden$="{{forgotPasswordError}}">[[forgotPasswordErrorMessage]]</span>
+      </div>
+      <div class="card-actions">
+        <paper-button on-tap="closeForgotPassword" raised>Cancel</paper-button>
+        <paper-button on-tap="sendForgotPassword" raised>Send email</paper-button>
+      </div>
+    </account-page>
+  </template>
+  <script>
+    Polymer({
+      is: 'forgot-password-request-page',
+      properties: {
+        selected: {
+          type: String,
+          notify: true
+        }
+      },
+      ready: function() {
+        this.forgotPasswordError = true;
+      },
+      sendForgotPassword: function() {
+        this.body = {
+          username: this.username.toLowerCase(),
+          email: this.email
+        };
+        this.$.account.generateRequest();
+      },
+      closeForgotPassword: function() {
+        this.selected = "";
+      },
+      handleForgotPasswordError: function(e) {
+        this.forgotPasswordError = false;
```

Comment:
I'd rename this to `displayForgotPasswordError`, I read this as you resetting/overriding the error